### DISCUSSION TITLE
[Question][Hw3] "keygen" description seems slightly misleading

### DISCUSSION THREAD

**Discussion by tncks0121** [prosyslab-classroom/cs348-information-security#694] [Q&A]
Name: Suchan Park

The description asks us to..

> generate **the two largest prime numbers** below the given upperbound.

So I understood I have to choose two distinct largest primes `p` and `q` **strictly smaller than** `ub`. (since, for example, 19 is not below 19)

But it seems I have to generate two distinct largest primes `p` and `q` **smaller than or equal to (no larger than)** `ub`.

If you get red verdict on Gradescope, try to change this. I thought I got red because my program is slow, but it seems the verdict is green if and only if outputs are all correct

**Comment by yeonhee-ryou** [comment]
You're correct. Thank you for the clarification. We will update the readme.
